backend/src/infrastructure/entities/language.entity.ts

```diff
@@ -0,0 +1,18 @@
+import { Entity, Column, PrimaryGeneratedColumn } from "typeorm";
+
+@Entity('language')
+export class LanguageEntity {
+  private _type = 'entity';
+
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Column()
+  name: string;
+
+  @Column()
+  nameEnglish: string;
+
+  @Column()
+  nameAlpha3: string;

```

Comment:
What is it used for?

Reply:
It is the ISO 639-2 code of the language

Reply:
ISO 639-1 is alpha2

Reply:
Ok fine!